openfl/component/aggregator/aggregator.py

```diff
@@ -20,6 +19,8 @@
 
 logger = logging.getLogger(__name__)
 
+VALID_MODES = {"train_and_validate", "evaluate"}
```

Comment:
I would rename the modes to `learning` and `evaluation` for consistency with common terminology (i.e. federated learning, federated evaluation, federated analytics, etc.)